src/components/pages/Profile/EditProfile.js

```diff
@@ -1,9 +1,8 @@
-import React, { useState, useEffect } from 'react';
```

Comment:
Cleaning up the code this way can help save storage and make the code easier to read.  Thanks!

Reply:
Errors like this are the bane of my existence. Thank you for removing this!